lib/matplotlib/backends/backend_svg.py

```diff
@@ -1026,15 +1026,10 @@ def _draw_text_as_text(self, gc, x, y, s, prop, angle, ismath, mtext=None):
             font = self._get_font(prop)
             font.set_text(s, 0.0, flags=LOAD_NO_HINTING)
 
-            fontsize = prop.get_size_in_points()
-
-            fontfamily = font.family_name
-            fontstyle = prop.get_style()
-
             attrib = {}
             # Must add "px" to workaround a Firefox bug
-            style['font-size'] = short_float_fmt(fontsize) + 'px'
-            style['font-family'] = str(fontfamily)
+            style['font-size'] = short_float_fmt(prop.get_size()) + 'px'
```

Comment:
`.get_size_in_points` and `.get_size` are methods with identical bodies....

Reply:
I know... planning to deprecate get_size_in_points but there are bugger issues with fontproperties anyways.